### ISSUE TITLE
Update to work with cake 0.26.0

### ISSUE THREAD

**Issue by sqeezy** [SharpeRAD/Cake.Powershell#47]
Using a fresh cakefile and adding the Cake.Powershell addin I get the following error:

```
Error: The assembly 'Cake.Powershell, Version=0.4.3.0, Culture=neutral, PublicKeyToken=null' 
is referencing an older version of Cake.Core (0.22.0). 
This assembly need to reference at least Cake.Core version 0.26.0. 
Another option is to downgrade Cake to an earlier version. 
It's not recommended, but you can explicitly opt-out of assembly verification 
by configuring the Skip Verification setting to true
(i.e. command line parameter "--settings_skipverification=true", 
environment variable "CAKE_SETTINGS_SKIPVERIFICATION=true", 
read more about configuration at https://cakebuild.net/docs/fundamentals/configuration)
```

**Comment by holytshirt** [comment]
This pull request should sort it out

**Comment by holytshirt** [comment]
https://github.com/SharpeRAD/Cake.Powershell/pull/48

**Comment by SharpeRAD** [comment]
Resolved in v0.4.4 👍
